### PR TITLE
Update the meta info again

### PR DESCRIPTION
**Pull request by kai-li-wop**
Some changes and additions to bring the content in line with other WoP profile pages. The goal is to unify the meta information.

### CODE REVIEW

File: net.worldofpadman.WoP.metainfo.xml
Comment (AsciiWolf) [inline net.worldofpadman.WoP.metainfo.xml:10]:
```suggestion
  <summary>Incredibly carefully designed and colorful freeware fun shooter for young and young-at-heart people</summary>
```

The summary is too long, should not contain the dot at the end and does not have to contain "World of PADMAN".

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

**Comment by AsciiWolf** [comment]
Looks good except the summary. :-) By the way, we probably could replace the currently used low-quality screenshots with better ones.

File: net.worldofpadman.WoP.metainfo.xml
Comment (AsciiWolf) [reply net.worldofpadman.WoP.metainfo.xml:10]: It looks like that for some reason I am not able to commit the suggestion. It is up to you, @kai-li-wop. :-)

I will merge the PR after the summary is fixed.

Comment (kai-li-wop) [reply net.worldofpadman.WoP.metainfo.xml:10]: Where can I see the restrictions of length? Is there a best practice documented somwhere? I don't have time right now, but I will check the upcoming days.

Comment (AsciiWolf) [reply net.worldofpadman.WoP.metainfo.xml:10]: I am not sure (the [official spec documentation](https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-summary) states only that it should be "A short summary of what this component does."), however this is from my experience as a software maintainer. I worked on many things including GNOME Software (software store GUI for GNOME) over the years and have also written dozens of AppStream metadata and desktop files, so I know the best practices.

Anyway, this is where the summary gets displayed:

summary

summary2

**Comment by kai-li-wop** [comment]
@AsciiWolf: I reviewed and applied your changes. Please try to merge again. For me merging is still shown as blocked.

**Review by AsciiWolf — APPROVED** [review]
Looks good now, thanks! :-)

**Pull request merged**
